guides/common/modules/proc_editing-the-system-purpose-of-a-host.adoc

```diff
@@ -2,7 +2,7 @@
 = Editing the System Purpose of a Host
 
 You can edit the system purpose attributes for a {RHEL} host.
-System purpose attributes define which subscriptions to attach automatically to this host.
+System purpose allows you to set the intended use of a system on your network and improves the reporting of usage in Subscription Watch.
```

Comment:
Per Jamie Prevatt, this text should now change to
```suggestion
System purpose enables you to set the system's intended use on your network and improves reporting accuracy in the Subscriptions service of the Red Hat Hybrid Cloud Console.
```

(sorry for the multiple edits! ping me if you want more context :)